src/unity.c

```diff
@@ -676,33 +673,28 @@ void UnityAssertEqualFloatArray(UNITY_PTR_ATTRIBUTE const UNITY_FLOAT* expected,
                                 const UNITY_LINE_TYPE lineNumber)
 {
     UNITY_UINT32 elements = num_elements;
-    UNITY_PTR_ATTRIBUTE const UNITY_FLOAT* ptr_expected = expected;
-    UNITY_PTR_ATTRIBUTE const UNITY_FLOAT* ptr_actual = actual;
```

Comment:
This additional assignment is here to make the PIC compilers happy. They require a particular attribute for some of the compilers to be assigned to the pointers, letting the compiler no they are a non-standard size.